README.md

```diff
@@ -632,6 +632,43 @@ Error inspecting states in the "s3" backend:
 `deployments/.active/.terraform/terraform.tfstate` refers to the correct bucket,
 the one configured in `AZUL_VERSIONED_BUCKET`. If it doesn't, you may
 have to remove that file or modify it to fix the bucket name.
+
+Or
```

Comment:
I think this needs to be placed in separate section since its a separate error, even though it occurs during `make deploy` like the previous one. It took me several minutes to understand why the explanation began with "Or".